gradle/libs.versions.toml

```diff
@@ -4,10 +4,10 @@
 androidCompileSdk = "33"
 androidMinSdk = "21"
 androidTargetSdk = "32"
-androidx-lifecycle = "2.4.1"
+androidx-lifecycle = "2.5.1"
```

Comment:
I don't suppose we could update this on 1.x as well? it's a blocker for adding dagger in a future PR
https://github.com/bumble-tech/appyx/pull/115

Reply:
I'll update now in a separate PR